internal/codespaces/rpc/invoker.go

```diff
@@ -20,38 +20,45 @@ import (
 
 const (
 	ConnectionTimeout = 5 * time.Second
-	RequestTimeout    = 30 * time.Second
+	requestTimeout    = 30 * time.Second
 )
 
 const (
 	codespacesInternalPort        = 16634
 	codespacesInternalSessionName = "CodespacesInternal"
 )
 
-type Client struct {
+type liveshareSession interface {
+	Close() error
+	GetSharedServers(context.Context) ([]*liveshare.Port, error)
+	KeepAlive(string)
+	OpenStreamingChannel(context.Context, liveshare.ChannelID) (ssh.Channel, error)
+	StartSharing(context.Context, string, int) (liveshare.ChannelID, error)
+	StartSSHServer(context.Context) (int, string, error)
+	StartSSHServerWithOptions(context.Context, liveshare.StartSSHServerOptions) (int, string, error)
+	RebuildContainer(context.Context, bool) error
+}
+
+type Invoker struct {
```

Comment:
Would be useful to make an interface around this from a testing standpoint so that it can be mocked?